app/components/preprint-status-banner/component.js

```diff
@@ -257,20 +260,29 @@ export default Component.extend({
             this.set('decision', this.get('submission.reviewsState'));
             this.set('reviewerComment', this.get('initialReviewerComment'));
         },
+        closeFeedback() {
+            if (!this.get('pendingStateNotChanged') || this.get('userActivity')) {
+                this.get('setUserEnteredReview')(this.get('userActivity'));
+            }
+        },
+        decisionToggled() {
+            this.set('decisionValueToggled', true);
+        },
     },
 
     _handleActions(action) {
         if (action) {
             if (this.get('submission.reviewsState') !== PENDING) {
                 const comment = action.get('comment');
-
                 this.set('initialReviewerComment', comment);
                 this.set('reviewerComment', comment);
                 this.set('decision', this.get('submission.reviewsState'));
+                this.set('initialDecision', this.get('submission.reviewsState'));
             } else {
                 this.set('initialReviewerComment', '');
                 this.set('reviewerComment', '');
                 this.set('decision', ACCEPTED);
+                this.set('initialDecision', PENDING);
```

Comment:
It looks like `initialDecision` is now just the same as `submission.reviewsState`, could we use that instead?